app/src/main/res/layout/village_typeahead_list_item.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:background="@color/white"
+  android:orientation="vertical">
+
+  <TextView
+    android:id="@+id/villageTypeAheadItemTextView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_vertical"
+    android:minHeight="?android:attr/listPreferredItemHeightSmall"
+    android:padding="@dimen/spacing_12"
+    android:textAppearance="?android:attr/textAppearanceListItemSmall"
```

Comment:
`textAppearance` for this TextView should be `textAppearanceBody1` based on the designs.

Reply:
I didn't find this specified in the designs can you point me where it was later? Thanks.

Reply:
In Figma the design team didn't seem to attach the text style, in those cases you should leave a comment on Figma tagging Pragati asking for the style or you can check the text styles in the app and see which matches the Figma values.

Reply:
👍🏻